### DISCUSSION TITLE
国内镜像由于未备案无法打开

### DISCUSSION THREAD

**Discussion by LouiseShariarc** [LmeSzinc/AzurLaneAutoScript#3053] [General]
国内镜像由于未备案无法打开，导致更新失败
![image](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/120891642/10ec2c73-a500-4ebf-9236-5528f17664f3)

**Comment by LmeSzinc** [comment]
更新服务不走80 443端口，也没有web服务，不需要备案
